qa/L0_config/config_generator.py

```diff
@@ -52,8 +52,8 @@ def _get_sweep_configs(profile_models):
             } for model in profile_models
         },
     }
-    # total_param=5 because the default config will also be generated
-    model_config['total_param'] = 5
+    # total_param=7 due to dynamic batching
```

Comment:
explain better.  5 from the previous run, and 4 configs here, but 2 are the same as the previous run

Reply:
Done - also updated based on the merge with main